neqo-bin/src/client/mod.rs

```diff
@@ -493,6 +493,14 @@ fn qlog_new(args: &Args, hostname: &str, cid: &ConnectionId) -> Res<NeqoQlog> {
     .map_err(Error::QlogError)
 }
 
+const fn unspecified_addr(addr: &SocketAddr) -> SocketAddr {
```

Comment:
It seems like you could include the port number in the signature.

I'm also unsure about the name.  This might be `local_addr_for(remote_addr, local_port)`.  The usage below can use `local_port: 0`.